dev/com.ibm.ws.smallrye.reactive.streams.operators.1.0/bnd.bnd

```diff
@@ -0,0 +1,40 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#    IBM Corporation 
+# This wraps the external JAR.
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: com.ibm.ws.smallrye.reactive.streams.operators.1.0; singleton:=true
+                     
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+Import-Package: \
+  *
+
+# We will whittle this down once it is working
+Export-Package: \
+  io.smallrye.reactive.streams; version="1.0.0", \
+  io.smallrye.reactive.streams.operators; version="1.0.0", \
+  io.smallrye.reactive.streams.spi; version="1.0.0", \
+  io.smallrye.reactive.streams.stages; version="1.0.0", \
+  io.smallrye.reactive.streams.utils; version="1.0.0", \
+
+Include-Resource: \
+  @${repo;com.ibm.ws.io.smallrye.reactive:smallrye-reactive-streams-operators;1.0.0}
```

Comment:
Change to the unmodified smallrye bundle

Reply:
edit done